main.go

```diff
@@ -29,8 +29,9 @@ import (
 )
 
 const (
-	extensionName = "vault-lambda-extension"
-	vaultLogLevel = "VAULT_LOG_LEVEL" // Optional, one of TRACE, DEBUG, INFO, WARN, ERROR, OFF
+	extensionName    = "vault-lambda-extension"
+	extensionVersion = "0.9.0"
```

Comment:
I think we discussed that the work to inject the version could be done in another Jira ticket. Maybe we can discuss it in backlog refinement next week? If we are going to hardcode it for now, we should probably also make sure to update the VLE Release wiki page to include steps to update this constant.